BuddySave/config.example.json

```diff
@@ -1,7 +1,9 @@
 ﻿{
   "CloudPath": "C:\\GoogleDrive\\BuddySave",
   "Session": {
-    "UserName": "buddy"
+    "UserName": "buddy",
+    "Ip": "127.0.0.1",
+    "Port": "2456"
   },
   "ServerParameters":{
     "Path": "C:\\Program Files\\Steam\\steamapps\\common\Valheim dedicated server\\start_headless_server.bat",
```

Comment:
Github shows an issue in server path. We have `\Valheim` instead of `\\Valheim`

Reply:
Totally forgot about it as I remember saw it when setting up the config myself. :D 
Fixed it